website/docs/migrating-from-v1-to-v2.md

```diff
@@ -125,17 +126,17 @@ No actions needed.
 
 Deprecated. We wrote a custom CSS framework for Docusaurus 2 called Infima which uses CSS variables for theming. The docs are not quite ready yet and we will update here when it is. To overwrite Infima' CSS variables, create your own CSS file (e.g. `./src/css/custom.css`) and import it globally by passing it as an option to `@docusaurus/preset-classic`:
 
-```diff
```

Comment:
Our current theme (palenight) doesnt support diff highlighting :( FYI

See https://github.com/reduxjs/redux-starter-kit/pull/226#issuecomment-546677060

Reply:
That's why it's nicer to change it to js with highlights ;)

Reply:
wondering if we can somehow author our own prism-theme.

So modifying 
https://github.com/FormidableLabs/prism-react-renderer/blob/master/src/themes/palenight.js

and call it `docusaurus prism theme` HAHA2

We can also scrap it from vscode theme using https://github.com/FormidableLabs/prism-react-renderer/tree/master/tools/themeFromVsCode